source/fundamentals/builders/aggregates.txt

```diff
@@ -50,6 +48,32 @@ aggregation as a list:
    :language: java
    :dedent:
 
+Documents
+---------
+
+Use the ``documents()`` method to create a
+:manual:`$documents </reference/operator/aggregation/documents/>`
+pipeline stage that returns literal documents from query values.
```

Comment:
Q: What is a "query value"? I noticed that the manual page refers to these as input values.

Reply:
That's...strange. I thought I copied the text directly from the manual. Changed.